fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirResourceTableGroup.java

```diff
@@ -239,6 +271,8 @@ public void addLogicalResources(List<IDatabaseObject> group, String prefix) {
      * V0010: IS_DELETED is added to each xxx_LOGICAL_RESOURCES.
      * V0011: LAST_UPDATED is added to each xxx_LOGICAL_RESOURCES.
      * V0012: VERSION_ID is added to each xxx_LOGICAL_RESOURCES.
+     * V0014: CANONICAL_ID is added to each xxx_LOGICAL_RESOURCES.
+     *        VERSION is added to each xxx_LOGICAL_RESOURCES.
```

Comment:
was this actually added?  i didn't see it in the migration logic below...

Reply:
I think that comment was from when I was considering a different design, but definitely doesn't apply now so I removed it